pino.js

```diff
@@ -306,9 +352,3 @@ module.exports.stdSerializers = {
   res: serializers.asResValue,
   err: serializers.asErrValue
 }
-module.exports.pretty = pretty
-Object.defineProperty(
-  module.exports,
-  'LOG_VERSION',
-  {value: LOG_VERSION, enumerable: true}
-)
```

Comment:
Apologies, this was not intentional, must be a bad case of mouse select + delete, and as it did not break the tests I did not spot it. I will put that block back in.